### ISSUE TITLE
Add a city history option in city info page

### ISSUE THREAD

**Issue by Swati4star** [project-travel-mate/Travel-Mate#380]
### Expected behaviour
- Add a new card `City History` in the city info page (just below the weather card)
- On clicking this, open up a new activity

The city history info has various levels of headings & under that subheadings (similar to wikipedia page : https://en.wikipedia.org/wiki/Delhi, checkout it's content section) [An example is given below]

On the city history activity, show the information received from the API. The headings should be bold.
Make each heading an expandable view (similar to city description in city info page)

### Response format
```
{
    "Summary": "Delhi (, Hindustani pronunciation: [d̪ɪlliː] Dilli), officially the National Capital Territory of Delhi (NCT), is a city and a union territory of India.It is bordered by Haryana on three sides and by Uttar Pradesh to the east. The NCT covers an area of 1,484 square kilometres (573 sq mi). According to the 2011 census, Delhi city proper's population was over 11 million, the second-highest in India after Mumbai, while the whole NCT's population was about 16.8 million. Delhi's urban area is now considered to extend beyond the NCT boundary and include the neighboring cities of Faridabad, Gurgaon, Noida and Ghaziabad in an area now called  Central National Capital Region (CNCR) with an estimated 2016 population of over 26 million people, making it the world's second-largest urban area according to United Nations.As of  2016 recent estimates of the metro economy of its urban area have ranked Delhi either the most or second-most productive metro area of India.Delhi is the second-wealthiest city after Mumbai in India, with a total wealth of $450 billion and home to 18 billionaires and 23,000 millionaires. Delhi is classed as a Alpha global city.Delhi has been continuously inhabited since the 6th century BC. Through most of its history, Delhi has served as a capital of various kingdoms and empires. It has been captured, ransacked and rebuilt several times, particularly during the medieval period, and modern Delhi is a cluster of a number of cities spread across the metropolitan region.A union territory, the political administration of the NCT of Delhi today more closely resembles that of a state of India, with its own legislature, high court and an executive council of ministers headed by a Chief Minister. New Delhi is jointly administered by the federal government of India and the local government of Delhi, and is the capital of the NCT of Delhi. Delhi hosted the first and ninth Asian Games in 1951 and 1982, respectively, 1983 NAM Summit, 2010 Men's Hockey World Cup, 2010 Commonwealth Games, 2012 BRICS Summit and was one of the major host cities of the 2011 Cricket World Cup.Delhi is also the centre of the National Capital Region (NCR), which is a unique 'interstate regional planning' area created by the National Capital Region Planning Board Act of 1985.",
    "Toponymy": "There are a number of myths and legends associated with the origin of the name Delhi. One of them is derived from Dhillu or Dilu, a king who built a city at this location in 50 BC and named it after himself. Another legend holds that the name of the city is based on the Hindi/Prakrit word dhili (loose) and that it was used by the Tomaras to refer to the city because the iron pillar of Delhi had a weak foundation and had to be moved. The coins in circulation in the region under the Tomaras were called dehliwal. According to the Bhavishya Purana, King Prithiviraja of Indraprastha built a new fort in the modern-day Purana Qila area for the convenience of all four castes in his kingdom. He ordered the construction of a gateway to the fort and later named the fort dehali. Some historians believe that the name is derived from Dilli, a corruption of the Hindustani words dehleez or dehali—both terms meaning 'threshold' or 'gateway'—and symbolic of the city as a gateway to the Gangetic Plain. Another theory suggests that the city's original name was Dhillika.The people of Delhi are referred to as Delhiites or Dilliwalas. The city is referenced in various idioms of the Northern Indo-Aryan languages. Examples include:Abhi Dilli door hai (अभी दिल्ली दूर है) or its Persian version, Hanuz Dehli dur ast (هنوز دلی دور است), literally meaning Delhi is still far away, which is generically said about a task or journey still far from completion.Dilli dilwalon ka shehr or Dilli Dilwalon ki meaning Delhi belongs to the large-hearted/daring.Aas-paas barse, Dilli pani tarse, literally meaning it pours all around, while Delhi lies parched. An allusion to the sometimes semi-arid climate of Delhi, it idiomatically refers to situations of deprivation when one is surrounded by plenty.",
    "History": "The area around Delhi was probably inhabited before the second millennium BC and there is evidence of continuous inhabitation since at least the 6th century BC. The city is believed to be the site of Indraprastha, the legendary capital of the Pandavas in the Indian epic Mahabharata. According to Mahabharata, this land was initially a huge mass of forests called 'Khandavaprastha' which was burnt down to build the city of Indraprastha. The earliest architectural relics date back to the Maurya period (c. 300 BC); in 1966, an inscription of the Mauryan Emperor Ashoka (273–235 BC) was discovered near Srinivaspuri. Remains of eight major cities have been discovered in Delhi. The first five cities were in the southern part of present-day Delhi. King Anang Pal of the Tomara dynasty founded the city of Lal Kot in AD 736. Prithviraj Chauhan conquered Lal Kot in 1178 and renamed it Qila Rai Pithora.The king Prithviraj Chauhan was defeated in 1192 by Muhammad Ghori, a Muslim invader from Afghanistan, who made a concerted effort to conquer northern India. By 1200, native Hindu resistance had begun to crumble, the dominance of foreign Turkic Muslim dynasties in north India was to last for the next five centuries. The slave general of Ghori, Qutb-ud-din Aibak was given the responsibility of governing the conquered territories of India and then Ghori returned to his capital, Ghor. He died in 1206 AD. He had no heirs and so his generals declared themselves independent in different parts of his empire. Qutb-ud-din assumed control of Ghori's Indian possessions. He laid the foundation of the Delhi Sultanate and the Mamluk dynasty. he began construction of the Qutb Minar and Quwwat-al-Islam (Might of Islam) mosque, the earliest extant mosque in India. It was his successor, Iltutmish (1211–36), who consolidated the Turkic conquest of northern India. Razia Sultan, daughter of Iltutmish, succeeded him as the Sultan of Delhi. She is the first and only woman to rule over Delhi.For the next three hundred years, Delhi was ruled by a succession of Turkic and an Afghan, Lodi dynasty. They built several forts and townships that are part of the seven cities of Delhi. Delhi was a major centre of Sufism during this period. The Mamluk Sultanate (Delhi) was overthrown in 1290 by Jalal ud din Firuz Khalji (1290–1320). Under the second Khalji ruler, Ala-ud-din Khalji, the Delhi sultanate extended its control south of the Narmada River in the Deccan. The Delhi sultanate reached its greatest extent during the reign of Muhammad bin Tughluq (1325–1351). In an attempt to bring the whole of the Deccan under control, he moved his capital to Daulatabad, Maharashtra in central India. However, by moving away from Delhi he lost control of the north and was forced to return to Delhi to restore order. The southern provinces then broke away. In the years following the reign of Firoz Shah Tughlaq (1351–1388), the Delhi Sultanate rapidly began to lose its hold over its northern provinces. Delhi was captured and sacked by Timur in 1398, who massacred 100,000 captives. Delhi's decline continued under the Sayyid dynasty (1414–1451), until the sultanate was reduced to Delhi and its hinterland. Under the Afghan Lodi dynasty (1451–1526), the Delhi sultanate recovered control of the Punjab and the Gangetic plain to once again achieve domination over Northern India. However, the recovery was short-lived and the sultanate was destroyed in 1526 by Babur, founder of the Mughal dynasty.Babur was a descendant of Genghis Khan and Timur, from the Fergana Valley in modern-day Uzbekistan. In 1526, he invaded India, defeated the last Lodhi sultan in the First Battle of Panipat and founded the Mughal Empire that ruled from Delhi and Agra. The Mughal dynasty ruled Delhi for more than three centuries, with a sixteen-year hiatus during the reigns of Sher Shah Suri and Hemu from 1540 to 1556. In 1553, the Hindu king Hemu acceded to the throne of Delhi by defeating forces of Mughal Emperor Humayun at Agra and Delhi. However, the Mughals re-established their rule after Akbar's army defeated Hemu during the Second Battle of Panipat in 1556. Shah Jahan built the seventh city of Delhi that bears his name Shahjahanabad, which served as the capital of the Mughal Empire from 1638 and is today known as the Old City or Old Delhi.After the death of Aurangzeb in 1707, the Mughal Empire's influence declined rapidly as the Hindu Maratha Empire from Deccan Plateau rose to prominence. In 1737, Maratha forces sacked Delhi following their victory against the Mughals in the First Battle of Delhi. In 1739, the Mughal Empire lost the huge Battle of Karnal in less than three hours against the numerically outnumbered but militarily superior Persian army led by Nader Shah of Persia. After his invasion, he completely sacked and looted Delhi, carrying away immense wealth including the Peacock Throne, the Daria-i-Noor, and Koh-i-Noor. The Mughals, severely further weakened, could never overcome this crushing defeat and humiliation which also left the way open for more invaders to come, including eventually the British. Nader eventually agreed to leave the city and India after forcing the Mughal emperor Muhammad Shah I to beg him for mercy and granting him the keys of the city and the royal treasury. A treaty signed in 1752 made Marathas the protectors of the Mughal throne in Delhi.In 1757, the Afghan ruler, Ahmad Shah Durrani, sacked Delhi. He returned to Afghanistan leaving a Mughal puppet ruler in nominal control. The Marathas again occupied Delhi in 1758, and were in control until their defeat in 1761 at the third battle of Panipat when the city was captured again by Ahmad Shah. However, in 1771, the Marathas established a protectorate over Delhi when the Maratha ruler, Mahadji Shinde, recaptured Delhi and the Mughal Emperor Shah Alam II was installed as a puppet ruler in 1772. In 1783, Sikhs under Baghel Singh captured Delhi and Red Fort but due to the treaty signed, Sikhs withdrew from Red Fort and agreed to restore Shah Alam II as the emperor. In 1803, during the Second Anglo-Maratha War, the forces of British East India Company defeated the Maratha forces in the Battle of Delhi.During the Indian Rebellion of 1857, Delhi fell to the forces of East India Company after a bloody fight known as the Siege of Delhi. The city came under the direct control of the British Government in 1858. It was made a district province of the Punjab. In 1911, it was announced that the capital of British held territories in India was to be transferred from Calcutta to Delhi. The name \"New Delhi\" was given in 1927, and the new capital was inaugurated on 13 February 1931. New Delhi, also known as Lutyens' Delhi, was officially declared as the capital of the Union of India after the country gained independence on 15 August 1947.During the partition of India, thousands of Hindu and Sikh refugees, mainly from West Punjab fled to Delhi, while many Muslim residents of the city migrated to Pakistan. Migration to Delhi from the rest of India continues (as of  2013), contributing more to the rise of Delhi's population than the birth rate, which is declining.The States Reorganisation Act, 1956 and the States Reorganisation Act, 1956 created the Union Territory of Delhi from the its predecessor the Chief Commissioner's Province of Delhi.The Constitution (Sixty-ninth Amendment) Act, 1991 declared the Union Territory of Delhi to be formally known as the National Capital Territory of Delhi. The Act gave Delhi its own legislative assembly along Civil lines, though with limited powers.In December 2001, the Parliament of India building in New Delhi was attacked by armed militants, killing six security personnel. India suspected Pakistan-based militant groups were behind the attack, which caused a major diplomatic crisis between the two countries. There were further terrorist attacks in Delhi in October 2005 and September 2008, resulting in a total of 103 deaths.",
    "Ecology": {
        "Summary": "Delhi is located at 28.61°N 77.23°E\ufeff / 28.61; 77.23, and lies in Northern India. It borders the Indian states of Haryana on the north, west and south and Uttar Pradesh (UP) to the east. Two prominent features of the geography of Delhi are the Yamuna flood plains and the Delhi ridge. The Yamuna river was the historical boundary between Punjab and UP, and its flood plains provide fertile alluvial soil suitable for agriculture but are prone to recurrent floods. The Yamuna, a sacred river in Hinduism, is the only major river flowing through Delhi. The Hindon River separates Ghaziabad from the eastern part of Delhi. The Delhi ridge originates from the Aravalli Range in the south and encircles the west, north-east and north-west parts of the city. It reaches a height of 318 m (1,043 ft) and is a dominant feature of the region.The National Capital Territory of Delhi covers an area of 1,484 km2 (573 sq mi), of which 783 km2 (302 sq mi) is designated rural, and 700 km2 (270 sq mi) urban therefore making it the largest city in terms of area in the country. It has a length of 51.9 km (32 mi) and a width of 48.48 km (30 mi).Delhi is included in India's seismic zone-IV, indicating its vulnerability to major earthquakes.",
        "Climate": "Delhi features an atypical version of the humid subtropical climate (Köppen Cwa) bordering a hot semi-arid climate (Köppen BSh). The warm season lasts from 21 March to 15 June with an average daily high temperature above 39 °C (102 °F). The hottest day of the year is 22 May, with an average high of 40 °C (104 °F) and low of 28 °C (82 °F). The cold season lasts from 26 November to 9 February with an average daily high temperature below 20 °C (68 °F). The coldest day of the year is 4 January, with an average low of 2 °C (36 °F) and high of 14 °C (57 °F). In early March, the wind direction changes from north-westerly to south-westerly. From April to October the weather is hot. The monsoon arrives at the end of June, along with an increase in humidity. The brief, mild winter starts in late November, peaks in January and heavy fog often occurs.Temperatures in Delhi usually range from 2 to 47 °C (35.6 to 116.6 °F), with the lowest and highest temperatures ever recorded being −2.2 and 48.4 °C (28.0 and 119.1 °F), respectively. The annual mean temperature is 25 °C (77 °F); monthly mean temperatures range from 13 to 32 °C (55 to 90 °F). The highest temperature recorded in July was 45 °C (113 °F) in 1931. The average annual rainfall is approximately 886 mm (34.9 in), most of which falls during the monsoon in July and August. The average date of the advent of monsoon winds in Delhi is 29 June.",
        "Air pollution": "According to the World Health Organization (WHO) Delhi was the most polluted city in the world in 2014. In 2016 WHO downgraded Delhi to eleventh-worst in the urban air quality database. According to one estimate, air pollution causes the death of about 10,500 people in Delhi every year. During 2013–14, peak levels of fine particulate matter (PM) in Delhi increased by about 44%, primarily due to high vehicular and industrial emissions, construction work and crop burning in adjoining states. It has the highest level of the airborne particulate matter, PM2.5 considered most harmful to health, with 153 micrograms. Rising air pollution level has significantly increased lung-related ailments (especially asthma and lung cancer) among Delhi's children and women. The dense smog in Delhi during winter season results in major air and rail traffic disruptions every year. According to Indian meteorologists, the average maximum temperature in Delhi during winters has declined notably since 1998 due to rising air pollution.Environmentalists have criticised the Delhi government for not doing enough to curb air pollution and to inform people about air quality issues. Most of Delhi's residents are unaware of alarming levels of air pollution in the city and the health risks associated with it; however, as of  2015, awareness, particularly among the foreign diplomatic community and high-income Indians, was noticeably increasing. Since the mid-1990s, Delhi has undertaken some measures to curb air pollution—Delhi has the third-highest quantity of trees among Indian cities and the Delhi Transport Corporation operates the world's largest fleet of environmentally friendly compressed natural gas (CNG) buses. In 1996, the Centre for Science and Environment (CSE) started a public interest litigation in the Supreme Court of India that ordered the conversion of Delhi's fleet of buses and taxis to run on compressed natural gas (CNG) and banned the use of leaded petrol in 1998. In 2003, Delhi won the United States Department of Energy's first 'Clean Cities International Partner of the Year' award for its \"bold efforts to curb air pollution and support alternative fuel initiatives\". The Delhi Metro has also been credited for significantly reducing air pollutants in the city.However, according to several authors, most of these gains have been lost, especially due to stubble burning, a rise in the market share of diesel cars and a considerable decline in bus ridership. According to CSE and System of Air Quality Weather Forecasting and Research (SAFAR), burning of agricultural waste in nearby Punjab, Haryana and Uttar Pradesh regions results in severe intensification of smog over Delhi. The state government of Uttar Pradesh is considering imposing a ban on crop burning to reduce pollution in Delhi NCR and an environmental panel has appealed to India's Supreme Court to impose a 30% cess on diesel cars.The Circles of Sustainability assessment of Delhi gives a marginally more favourable impression of the ecological sustainability of the city only because it is based on a more comprehensive series of measures than only air pollution. Part of the reason that the city remains assessed at basic sustainability is because of the low resource-use and carbon emissions of its poorer neighbourhoods."
    },
    "Civic administration": "As of July 2007, the National Capital Territory of Delhi comprises nine districts, 27 tehsils, 59 census towns, 300 villages, and three statutory towns, the Municipal Corporation of Delhi (MCD) – 1,397.3 km2 or 540 sq mi, the New Delhi Municipal Council (NDMC) – 42.7 km2 or 16 sq mi and the Delhi Cantonment Board (DCB) – 43 km2 or 17 sq mi).Since the trifurcation of the DMC at the start of 2012, Delhi has been run by five local municipal corporations: the North Delhi, South Delhi and East Delhi Municipal Corporations, the New Delhi Municipal Council and Delhi Cantonment Board. In July of that year, shortly after the MCD trifurcation, the Delhi Government increased the number of districts in Delhi from nine to eleven.Delhi (civic administration) was ranked 5th out of 21 Cities for best governance & administrative practices in India in 2014. It scored 3.6 on 10 compared to the national average of 3.3.Delhi houses the Supreme Court of India and the regional Delhi High Court along with the Small Causes Court for civil cases; the Magistrate Court and the Sessions Court for criminal cases has jurisdiction over Delhi. The city is administratively divided into eleven police-zones which are subdivided into 95 local police stations.",
    "Government and politics": "As a first-level administrative division, the National Capital Territory of Delhi has its own Legislative Assembly, Lieutenant Governor, council of ministers and Chief Minister. Members of the legislative assembly are directly elected from territorial constituencies in the NCT. The legislative assembly was abolished in 1956, after which direct federal control was implemented until it was re-established in 1993. The Municipal corporation handles civic administration for the city as part of the Panchayati Raj Act. The Government of India and the Government of National Capital Territory of Delhi jointly administer New Delhi, where both bodies are located. The Parliament of India, the Rashtrapati Bhavan (Presidential Palace), Cabinet Secretariat and the Supreme Court of India are located in the municipal district of New Delhi. There are 70 assembly constituencies and seven Lok Sabha (Indian parliament's lower house) constituencies in Delhi.The Indian National Congress (Congress) formed all the governments in Delhi until the 1990s, when the Bharatiya Janata Party (BJP), led by Madan Lal Khurana, came to power. In 1998, the Congress returned to power under the leadership of Sheila Dikshit, who was subsequently re-elected for 3 consecutive terms. But in 2013, the Congress was ousted from power by the newly formed Aam Aadmi Party (AAP) led by Arvind Kejriwal forming the government with outside support from the Congress. However, that government was short-lived, collapsing only after 49 days. Delhi was then under President's rule till February 2015. On 10 February 2015, the Aam Aadmi Party returned to power after a landslide victory, winning 67 out of the 70 seats in the Delhi Legislative Assembly.Since 2011 Delhi has three municipal bodiesSDMC having jurisdiction over South and West Delhi areas including Mahipalpur, Rajouri Garden, Janakpuri, Hari Nagar, Tilak Nagar, Dwarka, Jungpura, Greater Kailash, R K Puram, Malvya Nagar, Kalkaji, Ambedkar Nagar and Badarpur.NDMC has jurisdiction over areas such as Badli, Rithala, Bawana, Kirari, Mangolpuri, Tri nagar, Model Town, Sadar Bazar, Chandni Chowk, Matia Mahal, Karol Bagh, Moti NagarEDMC has jurisdiction over areas such as Patparganj, Kondli, Laxmi Nagar, Seemapuri, Gonda, Karawal Nagar, Babarpur and ShahadraIn 2017 BJP became victorious in all the three corporations",
    "Economy": "Delhi is the largest commercial centre in northern India. As of  2016 recent estimates of the economy of the Delhi urban area have ranged from $167 to $370 billion (PPP metro GDP) ranking it either the most or second-most productive metro area of India.The nominal GSDP of the NCT of Delhi for 2016-17 was estimated at ₹6,224 billion (US$91 billion), 13% higher than in 2015–16.As per the Economic survey of Delhi (2005–2006), the tertiary sector contributes 70.95% of Delhi's gross SDP followed by secondary and primary sectors with 25.20% and 3.85% contributions, respectively. Delhi's workforce constitutes 32.82% of the population, and increased by 52.52% between 1991 and 2001. Delhi's unemployment rate decreased from 12.57% in 1999–2000 to 4.63% in 2003. In December 2004, 636,000 people were registered with various employment exchange programmes in Delhi.In 2001 the total workforce in national and state governments and the quasi-government sector was 620,000, and the private sector employed 219,000. Key service industries are information technology, telecommunications, hotels, banking, media and tourism. Construction, power, health and community services and real estate are also important to the city's economy. Delhi has one of India's largest and fastest growing retail industries. Manufacturing also grew considerably as consumer goods companies established manufacturing units and headquarters in the city. Delhi's large consumer market and the availability of skilled labour has also attracted foreign investment. In 2001, the manufacturing sector employed 1,440,000 workers and the city had 129,000 industrial units.",
    "Utility services": "Delhi's municipal water supply is managed by the Delhi Jal Board (DJB). As of  June 2005, it supplied 650 million gallons per day (MGD), whereas the estimated consumption requirement is 963 MGD. The shortfall is met by private and public tube wells and hand pumps. At 240 MGD, the Bhakra storage is DJB's largest water source, followed by the Yamuna and Ganges rivers. Delhi's groundwater level is falling and its population density is increasing, so residents often encounter acute water shortage. Research on Delhi suggests that up to half of the city's water use is unofficial groundwater.In Delhi, daily domestic solid waste production is 8000 tonnes which is dumped at three landfill locations by MCD. The daily domestic waste water production is 470 MGD and industrial waste water is 70 MGD. A large portion of the sewage flows untreated into the Yamuna river.The city's electricity consumption is about 1,265 kWh per capita but the actual demand is higher. In Delhi power distribution is managed by Tata Power Distribution and BSES Yamuna & Rajdhani since 2002. The Delhi Fire Service runs 43 fire stations that attend about 15,000 fire and rescue calls per year. The state-owned Mahanagar Telephone Nigam Limited (MTNL) and private enterprises such as Vodafone, Airtel, Idea Cellular, Reliance Infocomm, Aircel, Reliance Jio and Tata Docomo provide telephone and cell phone services to the city. Cellular coverage is available in GSM, CDMA, 3G and 4G.",
    "Transport": {
        "Summary": "",
        "Air": "Indira Gandhi International Airport, situated to the southwest of Delhi, is the main gateway for the city's domestic and international civilian air traffic. In 2015–16, the airport handled more than 48 million passengers, making it the busiest airport in India and South Asia. Terminal 3, which cost ₹96.8 billion (US$1.4 billion) to construct between 2007 and 2010, handles an additional 37 million passengers annually.The Delhi Flying Club, established in 1928 with two de Havilland Moth aircraft named Delhi and Roshanara, was based at Safdarjung Airport which started operations in 1929, when it was the Delhi's only airport and the second in India. The airport functioned until 2001; however, in January 2002 the government closed the airport for flying activities because of security concerns following the New York attacks in September 2001. Since then, the club only carries out aircraft maintenance courses and is used for helicopter rides to Indira Gandhi International Airport for VIP including the president and the prime minister.A second airport open for commercial flights has been suggested either by expansion of Meerut Airport or construction of a new airport in Greater Noida.The Taj International Airport project in Jewar has been approved by the Uttar Pradesh government.",
        "Road": "Delhi has the highest road density of 2103 km/100 km2 in India. It is connected to other parts of India by five National Highways: NH 44, 2, 48, 9 and 24. The city's road network is maintained by MCD, NDMC, Delhi Cantonment Board, Public Works Department (PWD) and Delhi Development Authority.Buses are the most popular means of road transport catering to about 60% of Delhi's total demand. Delhi has one of India's largest bus transport systems. In 1998, the Supreme Court of India ruled that all public transport vehicles in Delhi must be fuelled by compressed natural gas (CNG) to tackle increasing vehicular pollution. The state-owned Delhi Transport Corporation (DTC) is a major bus service provider which operates the world's largest fleet of CNG-fuelled buses. In addition, cluster scheme buses are operated by Delhi Integrated Multi-Modal Transit System (DIMTS) with the participation of private concessionaires and DTC. In December 2017, the DTC and cluster buses carried over 4.19 million passengers per day. Kashmiri Gate ISBT, Anand Vihar ISBT and Sarai Kale Khan ISBT are the main bus terminals for outstation buses plying to neighboring states. Delhi's rapid rate of economic development and population growth has resulted in an increasing demand for transport, creating excessive pressure on the city's transport infrastructure. To meet the transport demand, the State and Union government constructed a mass rapid transit system, including the Delhi Metro. Delhi Bus Rapid Transit System runs between Ambedkar Nagar and Delhi Gate.  Personal vehicles especially cars also form a major chunk of vehicles plying on Delhi roads. As of  2007, private vehicles account for 30% of the total demand for transport. Delhi has the highest number of registered cars compared to any other metropolitan city in India. Taxis, auto rickshaws, and cycle rickshaws also ply on Delhi roads in large numbers. As of  2008, the number of vehicles in the metropolitan region, Delhi NCR, was 11.2 million (11.2 million). In 2008, there were 85 cars in Delhi for every 1,000 of its residents. In 2017, the number of vehicles in Delhi city alone crossed one crore mark with the transport department of Delhi Government putting the total number of registered vehicles at 1,05,67,712 till May 25 of the year.Important Roads in DelhiSome roads and expressways serve as important pillars of Delhi's road infrastructure:The Inner Ring Road is one of the most important \"state highways\" in Delhi. It is a 51 km long circular road which connects important areas in Delhi. Owing to more than 2 dozen grade-separators/flyovers, the road is almost signal-free.The Outer Ring Road is another major artery in Delhi that links far-flung areas of Delhi.The Delhi Noida Direct Flyway or DND Flyway is an eight-laned access controlled tolled expressway which connects Delhi to Noida (an important satellite city of Uttar Pradesh).The Delhi Gurgaon Expressway is a 28 km (17 mi) expressway connecting Delhi to Gurgaon, an important satellite city of Haryana.The Delhi Faridabad Skyway is controlled tolled expressway which connects Delhi to Faridabad, an important satellite city of Haryana.National Highways Passing Through DelhiDelhi is connected by road to various parts of the country through several National Highways:National Highway 44 (NH 44)the longest national highway starts from Srinagar and terminates in KanyakumariNational Highway 2 (NH 2) commonly referred as Delhi-Kolkata Road that runs through the states of Delhi, Haryana, Uttar Pradesh, Bihar, Jharkhand and West Bengal.National Highway 8 (NH 8) connects the Indian capital city of New Delhi with the Indian Financial capital city of Mumbai.National Highway 9 (NH 9) originates at Malaut in Punjab and ends at Askot in Uttarakhand.National Highway 24 (NH 24) connects the National capital Delhi to Uttar Pradesh state capital Lucknow running 438 kilometres (272 miles) in length.",
        "Railway": "Delhi is a major junction in the Indian railway network and is the headquarters of the Northern Railway. The main railway stations are New Delhi, Old Delhi, Hazrat Nizamuddin, Anand Vihar, Delhi Sarai Rohilla and Delhi Cantt. The Delhi Metro, a mass rapid transit system built and operated by Delhi Metro Rail Corporation (DMRC), serves many parts of Delhi and the neighbouring cities Faridabad, Gurgaon, Noida and Ghaziabad. As of August 2011, the metro consists of six operational lines with a total length of 189 km (117 mi) and 146 stations, and several other lines are under construction. The Phase-I was built at a cost of US$2.3 billion and the Phase-II was expected to cost an additional ₹216 billion (US$3.1 billion). Phase-II has a total length of 128 km and was completed by 2010. Delhi Metro completed 10 years of operation on 25 December 2012. It carries millions of passengers every day. In addition to the Delhi Metro, a suburban railway, the Delhi Suburban Railway exists.",
        "Metro": "The Delhi Metro is a rapid transit system serving Delhi, Faridabad, Gurgaon, Noida and Ghaziabad in the National Capital Region of India. Delhi Metro is the world's tenth-largest metro system in terms of length. Delhi Metro was India's second modern public transportation system, which has revolutionised travel by providing a fast, reliable, safe, and comfortable means of transport. The network consists of six lines with a total length of 189.63 kilometres (117.83 miles) with 142 stations, of which 35 are underground, five are at-grade, and the remainder are elevated. All stations have escalators, lifts, and tactile tiles to guide the visually impaired from station entrances to trains. There are 18 designated parking sites at Metro stations to further encourage use of the system. In March 2010, DMRC partnered with Google India (through Google Transit) to provide train schedule and route information to mobile devices with Google Maps.It has a combination of elevated, at-grade, and underground lines, and uses both broad gauge and standard gauge rolling stock. Four types of rolling stock are used: Mitsubishi-ROTEM Broad gauge, Bombardier MOVIA, Mitsubishi-ROTEM Standard gauge, and CAF Beasain Standard gauge. The Phase-I of Delhi Metro was built at a cost of US$2.3 billion and the Phase-II was expected to cost an additional ₹216 billion (US$3.1 billion). Phase-II has a total length of 128 km and was completed by 2010. Delhi Metro completed 10 years of operation on 25 December 2012. It carries millions of passengers every day.Delhi Metro is being built and operated by the Delhi Metro Rail Corporation Limited (DMRC), a state-owned company with equal equity participation from Government of India and Government of National Capital Territory of Delhi. However, the organisation is under the administrative control of Ministry of Urban Development, Government of India. Besides construction and operation of Delhi Metro, DMRC is also involved in the planning and implementation of metro rail, monorail, and high-speed rail projects in India and providing consultancy services to other metro projects in the country as well as abroad. The Delhi Metro project was spearheaded by Padma Vibhushan E. Sreedharan, the Managing Director of DMRC and popularly known as the \"Metro Man\" of India. He famously resigned from DMRC taking moral responsibility for a metro bridge collapse, which took five lives. Sreedharan was awarded the prestigious Legion of Honour by the French Government for his contribution to Delhi Metro.Metro services are being extended to important hubs in the cities that are close to offices, colleges, and tourist spots. This will facilitate easy conveyance for the citizens, who otherwise have to rely on public buses that are heavily crowded and are often stuck in traffic jams.",
        "Regional Rapid Transit System (RRTS)": "The 08 RRTS Corridors have been proposed by National Capital Region Planning Board (NCRPB) to facilitate the people travelling from nearby cities in NCR to Delhi. The three main corridors in the first phase are as follows:Delhi – Alwar via Gurugram - 180.50 kmDelhi – Panipat via Sonipat - 111 kmDelhi – Meerut via Ghaziabad - 92.05 kmRemaining five corridors are also approved by National Capital Region Planning Board but are planned in the second phase."
    },
    "Demographics": {
        "Summary": "According to the 2011 census of India, the population of NCT of Delhi is 16,753,235. The corresponding population density was 11,297 persons per km2 with a sex ratio of 866 women per 1000 men, and a literacy rate of 86.34%. In 2004, the birth rate, death rate and infant mortality rate per 1000 population were 20.03, 5.59 and 13.08, respectively. In 2001, the population of Delhi increased by 285,000 as a result of migration and by 215,000 as a result of natural population growth, which made Delhi one of the fastest growing cities in the world. Dwarka Sub City, Asia's largest planned residential area, is located within the National Capital Territory of Delhi.Urban expansion has resulted in Delhi's urban area now being considered as extending beyond NCT boundaries to incorporate towns and cities of neighbouring states including Gurgaon and Faridabad of Haryana, and Ghaziabad and Noida of Uttar Pradesh, the total population estimated by the United Nations at over 26 million. According to the UN this makes Delhi urban area the world's second-largest, after Tokyo, although Demographia declares the Jakarta urban area to be the second-largest. The 2011 census provided two figures for urban area population: 16,314,838 within the NCT boundary, and 21,753,486 for the Extended Urban Area. The 2021 regional plan released by the Government of India renamed the Extended Urban Area from Delhi Metropolitan Area (DMA) as defined by the 2001 plan to Central National Capital Region (CNCR).",
        "Religion": "Hinduism is Delhi's predominant religious faith, with 81.68% of Delhi's population, followed by Islam (12.86%), Sikhism (3.40%), Jainism (0.99%), Christianity (0.87%), and Buddhism (0.11%). Other minority religions include Zoroastrianism, Baha'ism and Judaism.",
        "Languages": "According to the 50th report of the commissioner for linguistic minorities in India, which was submitted in 2014, Hindi is Delhi's most spoken language, with 80.94% speakers, followed by Punjabi (7.14%), Urdu (6.31%) and Bengali (1.50%). 4.11% of the Delhites speak other languages. Hindi is also the official language of Delhi while Urdu and Punjabi have been declared as the additional official languages.According to the Directorate of Education, GNCTD the following languages are taught in schools in Delhi under the three-language formula:First Language: Hindi/Urdu/EnglishSecond Language: EnglishThird language: Urdu/Punjabi/Bengali/Sindhi/Tamil/Telugu/Malayalam/Kannada/Gujarati/Marathi/Arabic/Persian/Sanskrit",
        "Slums": "Around 49% of the population of Delhi lives in slums and unauthorized colonies without any civic amenities. Majority of the slums have inadequate provisions to the basic facilities and according to a DUSIB report, almost 22% of the people do open defecation."
    },
    "Culture": {
        "Summary": "Delhi's culture has been influenced by its lengthy history and historic association as the capital of India, Although a strong Punjabi Influence can be seen in language, Dress and Cuisine brought by the large number of refugees who came following the partition in 1947 the recent migration from other parts of India has made it a melting pot. This is exemplified by many significant monuments in the city. Delhi is also identified as the location of Indraprastha, the ancient capital of the Pandavas. The Archaeological Survey of India recognises 1,200 heritage buildings and 175 monuments as national heritage sites. In the Old City, the Mughals and the Turkic rulers constructed several architecturally significant buildings, such as the Jama Masjid—India's largest mosque built in 1656 and the Red Fort. Three World Heritage Sites—the Red Fort, Qutab Minar and Humayun's Tomb—are located in Delhi. Other monuments include the India Gate, the Jantar Mantar—an 18th-century astronomical observatory—and the Purana Qila—a 16th-century fortress. The Laxminarayan temple, Akshardham temple, Gurudwara Bangla Sahib, the Bahá'í Lotus Temple and the ISKCON temple are examples of modern architecture. Raj Ghat and associated memorials houses memorials of Mahatma Gandhi and other notable personalities. New Delhi houses several government buildings and official residences reminiscent of British colonial architecture, including the Rashtrapati Bhavan, the Secretariat, Rajpath, the Parliament of India and Vijay Chowk. Safdarjung's Tomb is an example of the Mughal gardens style. Some regal havelis (palatial residences) are in the Old City.Lotus Temple, is a Bahá'í House of Worship completed in 1986. Notable for its flowerlike shape, it serves as the Mother Temple of the Indian subcontinent and has become a prominent attraction in the city. The Lotus Temple has won numerous architectural awards and been featured in hundreds of newspaper and magazine articles. Like all other Bahá'í Houses of Worship, is open to all regardless of religion, or any other distinction, as emphasised in Bahá'í texts. The Bahá'í laws emphasise that the spirit of the House of Worship be that it is a gathering place where people of all religions may worship God without denominational restrictions. The Bahá'í laws also stipulate that only the holy scriptures of the Bahá'í Faith and other religions can be read or chanted inside in any language; while readings and prayers can be set to music by choirs, no musical instruments can be played inside. Furthermore, no sermons can be delivered, and there can be no ritualistic ceremonies practised.Chandni Chowk, a 17th-century market, is one of the most popular shopping areas in Delhi for jewellery and Zari saris. Delhi's arts and crafts include, Zardozi—an embroidery done with gold thread— and Meenakari—the art of enamelling.",
        "Festivals": "Delhi's association and geographic proximity to the capital, New Delhi, has amplified the importance of national events and holidays like Republic Day, Independence Day (15 August) and Gandhi Jayanti. On Independence Day, the Prime Minister addresses the nation from the Red Fort. Most Delhiites celebrate the day by flying kites, which are considered a symbol of freedom. The Republic Day Parade is a large cultural and military parade showcasing India's cultural diversity and military strength. Over the centuries, Delhi has become known for its composite culture, and a festival that symbolises this is the Phool Walon Ki Sair, which takes place in September. Flowers and pankhe—fans embroidered with flowers—are offered to the shrine of the 13th-century Sufi saint Khwaja Bakhtiyar Kaki and the Yogmaya temple, both situated in Mehrauli.Religious festivals include Diwali (the festival of lights), Mahavir Jayanti, Guru Nanak's Birthday, Raksha Bandhan, Durga Puja, Holi, Lohri, Chauth, Krishna Janmastami, Maha Shivratri, Eid ul-Fitr, Moharram and Buddha Jayanti. The Qutub Festival is a cultural event during which performances of musicians and dancers from all over India are showcased at night, with the Qutub Minar as a backdrop. Other events such as Kite Flying Festival, International Mango Festival and Vasant Panchami (the Spring Festival) are held every year in Delhi. The Auto Expo, Asia's largest auto show, is held in Delhi biennially. The New Delhi World Book Fair, held biennially at the Pragati Maidan, is the second-largest exhibition of books in the world. Delhi is often regarded as the \"Book Capital\" of India because of high readership. India International Trade Fair (IITF), organised by ITPO is the biggest cultural and shopping fair of Delhi which takes place in November each year and is visited by more than 15 lakh people.",
        "Cuisine": "As India's national capital and centuries old Mughal capital, Delhi influenced the food habits of its residents and is where Mughlai cuisine originated. Along with Indian cuisine, a variety of international cuisines are popular among the residents. The dearth of food habits among the city's residents created a unique style of cooking which became popular throughout the world, with dishes such as Kebab, biryani, tandoori. The city's classic dishes include butter chicken, dal makhani, shahi paneer, aloo chaat, chaat, dahi bhalla, kachori, gol gappe, samosa, chole bhature, chole kulche, gulab jamun, jalebi and lassi.The fast living habits of Delhi's people has motivated the growth of street food outlets. A trend of dining at local dhabas is popular among the residents. High-profile restaurants have gained popularity in recent years, among the popular restaurants are the Karim Hotel, the Punjab Grill and Bukhara. The Gali Paranthe Wali (the street of fried bread) is a street in Chandni Chowk particularly for food eateries since the 1870s. Almost the entire street is occupied by fast food stalls or street vendors. It has nearly become a tradition that almost every prime minister of India has visited the street to eat paratha at least once. Other Indian cuisines are also available in this area even though the street specialises in north Indian food ."
    },
    "Tourism": "  According to Euromonitor International, Delhi ranked as 28th-most visited city in the world and first in India by foreign visitors in 2015. There are numerous tourist attractions in Delhi, both historic and modern. The three UNESCO World Heritage Sites in Delhi, Qutb Complex, Red Fort and Humayun's Tomb are among the finest examples of Indo-Islamic architecture. Another prominent landmark of Delhi is India Gate, a 1931 built war memorial to soldiers of British Indian Army who died during First World War. Delhi has several famous places of worship of various religions. One of the largest Hindu temple complexes in the world, Akshardham is a major tourist attraction in the city. Other famous religious sites include Lal Mandir, Laxminarayan Temple, Gurudwara Bangla Sahib, Lotus Temple, Jama Masjid and ISKCON Temple. Delhi is also a hub for shopping of all kinds. Connaught Place, Chandni Chowk, Khan Market and Dilli Haat are some of the major retail markets in Delhi. Major shopping malls include Select Citywalk, DLF Promenade, DLF Emporio, Metro Walk and Ansal Plaza.",
    "Education": "Private schools in Delhi—which use either English or Hindi as the language of instruction—are affiliated to one of three administering bodies, the Council for the Indian School Certificate Examinations (CISCE), the Central Board for Secondary Education (CBSE) or the National Institute of Open Schooling (NIOS). In 2004–05, approximately 15.29 lakh (1.529 million) students were enrolled in primary schools, 8.22 lakh (0.822 million) in middle schools and 6.69 lakh (0.669 million) in secondary schools across Delhi. Female students represented 49% of the total enrolment. The same year, the Delhi government spent between 1.58% and 1.95% of its gross state domestic product on education.  Schools and higher educational institutions in Delhi are administered either by the Directorate of Education, the NCT government or private organisations. In 2006, Delhi had 165 colleges, five medical colleges and eight engineering colleges, seven major universities and nine deemed universities.The premier management colleges of Delhi such as Faculty of Management Studies (Delhi) and Indian Institute of Foreign Trade rank the best in India. All India Institute of Medical Sciences Delhi is a premier medical school for treatment and research. National Law University, Delhi is a prominent law school and is affiliated to the Bar Council of India.Delhi Technological University (formerly Delhi College of Engineering), Indraprastha Institute of Information Technology, Netaji Subhas Institute of Technology, Guru Gobind Singh Indraprastha University and National Law University, Delhi are the only state universities. University of Delhi, Jawaharlal Nehru University and Jamia Millia Islamia are the central universities, and Indira Gandhi National Open University is for distance education.As of  2008, about 16% of all Delhi residents possessed at least a college graduate degree.",
    "Media": "As the capital of India, Delhi is the focus of political reportage, including regular television broadcasts of Parliament sessions. Many national media agencies, including the state-owned Press Trust of India, Media Trust of India and Doordarshan, is based in the city. Television programming includes two free terrestrial television channels offered by Doordarshan, and several Hindi, English, and regional-language cable channels offered by multi system operators. Satellite television has yet to gain a large quantity of subscribers in the city.Print journalism remains a popular news medium in Delhi. The city's Hindi newspapers include Navbharat Times, Hindustan Dainik, Punjab Kesari, Pavitra Bharat, Dainik Jagran, Dainik Bhaskar, Amar Ujala and Dainik Desbandhu. Amongst the English language newspapers, The Hindustan Times, with a daily circulation of over a million copies, is the single largest daily. Other major English newspapers include Times of India, The Hindu, Indian Express, Business Standard, The Pioneer, The Statesman, and The Asian Age. Regional language newspapers include the Malayalam daily Malayala Manorama and the Tamil dailies Dinamalar and Dinakaran.Radio is a less popular mass medium in Delhi, although FM radio has gained popularity since the inauguration of several new stations in 2006.A number of state-owned and private radio stations broadcast from Delhi.",
    "Sports": "Delhi has hosted many major international sporting events, including the first and also the ninth Asian Games, the 2010 Hockey World Cup, the 2010 Commonwealth Games and the 2011 Cricket World Cup. Delhi lost bidding for the 2014 Asian Games, and considered making a bid for the 2020 Summer Olympics. However, sports minister Manohar Singh Gill later stated that funding infrastructure would come before a 2020 bid. There are indications of a possible 2028 bid.The 2010 Commonwealth Games, which ran from 3 to 14 October 2010, was one of the largest sports event held in India. The opening ceremony of the 2010 Commonwealth Games was held at the Jawaharlal Nehru Stadium, the main stadium of the event, in New Delhi at 7:00 pm Indian Standard Time on 3 October 2010. The ceremony featured over 8,000 performers and lasted for two and a half hours. It is estimated that ₹3.5 billion (US$51 million) were spent to produce the ceremony. Events took place at 12 competition venues. 20 training venues were used in the Games, including seven venues within Delhi University. The rugby stadium in Delhi University North Campus hosted rugby games for Commonwealth Games. The mess left behind after the Commonwealth Games prompted Prime Minister Manmohan Singh to replace Sports and Youth Affairs minister Manohar Singh Gill with Ajay Maken in 19 January 2011 Cabinet reshuffle.Cricket and football are the most popular sports in Delhi. There are several cricket grounds, or maidans, located across the city. The Feroz Shah Kotla Ground (known commonly as the Kotla) is one of the oldest cricket grounds in India and is a venue for international cricket matches. It is the home ground of the Delhi cricket team, which represents the city in the Ranji Trophy, the premier Indian domestic first-class cricket championship. The Delhi cricket team has produced several world-class international cricketers such as Virender Sehwag, Virat Kohli, Gautam Gambhir, Madan Lal, Chetan Chauhan, Ishant Sharma and Bishan Singh Bedi to name a few. The Railways and Services cricket teams in the Ranji Trophy also play their home matches in Delhi, in the Karnail Singh Stadium and the Harbax Singh Stadium, respectively. The city is also home to the Indian Premier League team Delhi Daredevils, who play their home matches at the Kotla.Ambedkar Stadium, a football stadium in Delhi which holds 21,000 people, was the venue for the Indian football team's World Cup qualifier against UAE on 28 July 2012. Delhi hosted the Nehru Cup in 2007 and 2009, in both of which India defeated Syria 1–0. In the Elite Football League of India, Delhi's first professional American football franchise, the Delhi Defenders played its first season in Pune. Buddh International Circuit in Greater Noida, a suburb of Delhi, formerly hosted the Formula 1 Indian Grand Prix. The Indira Gandhi Arena is also in Delhi.Delhi is a member of the Asian Network of Major Cities 21.Current Regional and Professional Sports Teams from DelhiFormer Regional and Professional Sports Teams from Delhi",
    "World Heritage status": "In February 2014, the Government of India approved Delhi's bid for World Heritage City status. The historical city of Shahjahanabad and Lutyens' Bungalow Zone in New Delhi were cited in the bid. A team from UNESCO was scheduled to visit Delhi in September 2014 to validate its claims. INTACH acted as the nodal agency for the bid. The announcement of accepted cities was to be made in June 2015. However, the Government of India withdrew its nomination on 21 May 2015.",
    "Notable people": "",
    "International relations": "Sister cities New York City, United States London, United Kingdom Moscow, Russia Ulaanbaatar, Mongolia Chicago, Illinois, United States Kuala Lumpur, Malaysia Tokyo, Japan Seoul, South Korea Fukuoka, Japan Kabul, Afghanistan (proposed 2017)Partnerships Yerevan, Armenia"
}
```

### Endpoint to use
`get-city-information/<city-id>`

**Comment by aayushinigam** [comment]
Would like to work on this.

**Comment by aayushinigam** [comment]
I'm finding this hard to implement. So I would prefer to work on some other issue for now.

**Comment by Swati4star** [comment]
@aayushinigam Sure. Pick up some other issue.
( But if you want to pick it up later, you can comment 👍 )

**Comment by aayushinigam** [comment]
Thanks @Swati4star :) .If I figure it out, I'll work on this.

**Comment by Swati4star** [comment]
@aayushinigam You can pick this up. Don't add expandable views for now. Let it just be a single static page, with headings (in bold) and data under it

**Comment by aayushinigam** [comment]
Oh okay!

**Comment by aayushinigam** [comment]
@Swati4star  So here's the thing. The above format is not exactly the same for all cities. In some cities, the given key exists, for some it doesn't. And hence the layout doesn't get filled. Had response been in JSONArray format , it would have been still easier to check through index. But it's a JSONObject, so keys have to be mentioned explicitly.

**Comment by aayushinigam** [comment]
Here's an example - 
![screenshot from 2018-08-02 21-07-52](https://user-images.githubusercontent.com/24635221/43594998-3db86030-9699-11e8-8212-0da40866dd52.png)

**Comment by Swati4star** [comment]
@aayushinigam True. The keys are not the same for all cities. So, you can use this code to get a list of keys from the JSON Object : https://stackoverflow.com/questions/20419217/to-get-all-the-keys-in-jsonobject-into-string-array
and then, keep on extracting values from it.

**Comment by aayushinigam** [comment]
@Swati4star Any suggestions for the UI?

![text](https://user-images.githubusercontent.com/24635221/43656764-3cbf48dc-9771-11e8-8f2e-f238ebef3b44.png)

**Comment by Swati4star** [comment]
- What about adding subheadings also? For example, in `Ecology`, we have `summary`.
- Also, in this view, there are `...` at the end of each section. How can the user view the whole content?

**Comment by aayushinigam** [comment]
The problem right now is that we don't know what keys will have sub headings plus it will then become multi level list view.
User can view whole content by clicking on text, it's an expandable text view.

**Comment by Swati4star** [comment]
oh okay. Looks good then, you can commit this.
